### ISSUE TITLE
False positive CVE-2018-12070 matches org.sonatype.plexus:plexus-sec-dispatcher:1.4

### ISSUE THREAD

**Issue by nomis** [dependency-check/DependencyCheck#1453]
False positive on library plexus-sec-dispatcher-1.4.jar - reported as cpe:/a:sec_project:sec:1.4
```xml
<dependency>
   <groupId>org.sonatype.plexus</groupId>
   <artifactId>plexus-sec-dispatcher</artifactId>
   <version>1.4</version>
</dependency>
```

**Comment by jeremylong** [comment]
Thanks - I'll make a pass through the FP reports before the next release.

**Comment by jeremylong** [comment]
fixed with #1485.
